package logica.excepciones;

public class UsuarioException extends Exception{
    public UsuarioException(){super("Datos inv√°lidos, probar nuevamente. Gracias");}
}
